core/trino-main/src/main/java/io/trino/server/remotetask/HttpRemoteTask.java

```diff
@@ -714,6 +715,16 @@ boolean adjustSplitBatchSize(List<SplitAssignment> splitAssignments, long reques
     }
 
     private void sendUpdate()
+    {
+        try {
+            sendUpdateUnsafe();
+        }
+        catch (Throwable e) {
+            fatalUnacknowledgedFailure(new TrinoException(GENERIC_INTERNAL_ERROR, "unexpected error calling sendUpdate()", e));
+        }
+    }
+
+    private void sendUpdateUnsafe()
```

Comment:
Why "unsafe"? Maybe call it "sendUpdateInternal"?

Reply:
Sure